.gitlab-ci.yml

```
@@ -25,12 +25,44 @@ stages:
   - deploy
   - notify
 
+########################################################################################################################
+# Configuration helpers
+########################################################################################################################
+
+.base-configuration:
+  tags: ['runner:main', 'size:large']
+  image: $CI_IMAGE
+
+.feature-branch-configuration:
+  extends: .base-configuration
+  except:
+    refs:
+      - main
+      - tags
+      - /^staging-[0-9]+$/
+      - /^release\/v[0-9.]+$/
```

```suggestion
      - /^release\//
```
let's be a bit more permissive on the release branch name
By looking at my history, I have some `release/X.Y.Z` branches and I don't think we need to enforce the `release/vX.Y.Z` format

---

Right